### ISSUE TITLE
aws-shell fails at startup on Darwin

### ISSUE THREAD

**Issue by copumpkin** [NixOS/nixpkgs#17326]
## Issue description

`aws-shell` crashes at startup.

### Steps to reproduce

```
$ nix-shell -p pythonPackages.aws_shell


[nix-shell:~]$ aws-shell
First run, creating autocomplete index...
Traceback (most recent call last):
  File "/nix/store/3gb6mxk35axmzp3x7dwnvfwxvh0s9afy-python2.7-aws-shell-0.1.0/bin/.aws-shell-wrapped", line 12, in <module>
    sys.exit(main())
  File "/nix/store/3gb6mxk35axmzp3x7dwnvfwxvh0s9afy-python2.7-aws-shell-0.1.0/lib/python2.7/site-packages/awsshell/__init__.py", line 47, in main
    write_index(index_file)
  File "/nix/store/3gb6mxk35axmzp3x7dwnvfwxvh0s9afy-python2.7-aws-shell-0.1.0/lib/python2.7/site-packages/awsshell/makeindex.py", line 63, in write_index
    index_command(current, help_command)
  File "/nix/store/3gb6mxk35axmzp3x7dwnvfwxvh0s9afy-python2.7-aws-shell-0.1.0/lib/python2.7/site-packages/awsshell/makeindex.py", line 55, in index_command
    index_command(child, sub_help_command)
  File "/nix/store/3gb6mxk35axmzp3x7dwnvfwxvh0s9afy-python2.7-aws-shell-0.1.0/lib/python2.7/site-packages/awsshell/makeindex.py", line 55, in index_command
    index_command(child, sub_help_command)
  File "/nix/store/3gb6mxk35axmzp3x7dwnvfwxvh0s9afy-python2.7-aws-shell-0.1.0/lib/python2.7/site-packages/awsshell/makeindex.py", line 43, in index_command
    arg, arg_obj.argument_model)
TypeError: generate_shorthand_example() takes exactly 4 arguments (3 given)

[nix-shell:~]$ 
```

## Technical details

* System: Mac OS
* Nix version: nix-env (Nix) 1.11.1
* Nixpkgs version: 16.09pre87131.292bfed

**Comment by FRidh** [comment]
isn't that fixed in 0.1.1?
https://github.com/awslabs/aws-shell/issues/118
cc @cransom

**Comment by FRidh** [comment]
Yes, its fixed on master fdd84fd86dc3c63db7d2b54b49958bf26c6eb69a
